CHANGES.rst

```diff
@@ -10,6 +10,7 @@ Version 0.9.0 (UNRELEASED)
 - Adds REANA specification validation utilities.
 - Changes API rate limiter error messages to be more verbose.
 - Changes workflow scheduler to allow defining the checks needed to assess whether the cluster can start new workflows.
+- Changes to the latest versions of Invenio dependencies.
```

Comment:
What do you think about `Updates the Invenio dependencies to the latest versions`?

Reply:
I agree that your suggestion sounds better, but there is a convention for the changelog to only use `changes/adds/fixes` as a prefix. @tiborsimko, I guess we should document it in REANA wiki?